src/data_downloads/ncbi_datasets_cli.md

```diff
@@ -0,0 +1,25 @@
+# ncbi-datasets: gathering data across NCBI databases
+
+[ncbi-datasets](https://github.com/ncbi/datasets) is a very-well documented open source tool from NCBI (National Center for Biotechnology Information) for gathering various kinds of data including genes, genomes, annotations etc. belonging to the desired species.
+As a side note, it has specific options for retrieving SARS-CoV-2 data.
+
+## Installation
+
+It is offered as a conda package and can be downloaded via:
+
+```
+conda install -c conda-forge ncbi-datasets-cli
```

Comment:
I usually try to work with isolated and minimal conda environments, so I'd suggest to also make this the default here:
```suggestion
conda create -n ncbi-download -c conda-forge ncbi-datasets-cli
```